### PR TITLE
Clean gates/signatures, remove comments, add gate

### PR DESCRIPTION
**Pull request by nunezco2**

### CODE REVIEW

File: src/qc_ncsa.aplf [hunk 1]
```diff
@@ -152,26 +152,52 @@
...
+        }
 
         ⍝ Ry (rotation in y)
-        Ry ← {2 2 ⍴ ((2○⍵÷2),(¯1×(1○⍵÷2)),(1○⍵÷2),(2○⍵÷2))}
+        Ry ← {
+            a ← 2○⍵÷2
+            b ← 1○⍵÷2
+            bm ← ¯1×b
+            2 2 ⍴ a bm b a
+        }
```
Comment (marcosfrenkel) [inline src/qc_ncsa.aplf:176]: what are the 1s and 0s under the values here: 
image

Comment (marcosfrenkel) [reply src/qc_ncsa.aplf:176]: is it an identity matrix?

File: src/qc_ncsa.aplf [hunk 2]
```diff
@@ -152,26 +152,52 @@
...
 
         ⍝ Rz (rotation in z)
-        Rz ← {2 2 ⍴ ((*¯0J1×(⍵÷2)),0,0,(*0J1×(⍵÷2)))}
+        Rz ← {
+            am ← *0J¯1×⍵÷2
+            ap ← *0J1×⍵÷2
+            2 2 ⍴ am 0 0 ap
+        }
```
Comment (marcosfrenkel) [inline src/qc_ncsa.aplf:183]: Same here, the return value changes from before:

image

Comment (nunezco2) [reply src/qc_ncsa.aplf:183]: Idem.

File: src/qc_ncsa.aplf (hunk 1 above)
Comment (marcosfrenkel) [reply src/qc_ncsa.aplf:176]: oh, you have to pass them the angle of the rotation? and that makes the specific gate for that specific rotation?

**Review by marcosfrenkel — APPROVED** [review]
Looks good!

**Pull request merged**